.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This should probably be in your global gitignore, it is not feasible for projects to keep track of every editor or development environment.